h5p.classes.php

```diff
@@ -1571,7 +1585,7 @@ public function createExportFile($content) {
 
     // Create new zip instance.
     $zip = new ZipArchive();
-    $zip->open($tmpFile, ZIPARCHIVE::CREATE | ZIPARCHIVE::OVERWRITE);
```

Comment:
This throws a warning, ZipArchive (as used above) is the correct class definition